packages/react-filerobot-image-editor/src/actions/selectTool.js

```
@@ -1,12 +1,13 @@
-    ? state
-    : {
-        ...state,
-        toolId: payload.toolId,
-        selectionsIds: payload.keepSelections ? state.selectionsIds : [],
-      };
+  ? state
+  : {
+      ...state,
+      toolId: payload.toolId,
+      selectionsIds: payload.keepSelections ? state.selectionsIds : [],
+    };
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(no-unused-expressions)